src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -508,7 +509,8 @@ static AppleBundle createAppleBundle(
         Optional<FrameworkDependencies> frameworkDependencies =
             resolver.requireMetadata(
                 BuildTarget.builder(dep)
-                    .addFlavors(FRAMEWORK_FLAVOR)
+                    //this seems wrong
```

Comment:
Yes, I think we'll need some way of saying "by default everything frameworks or libraries", but it should be overridable.

What would you say is a better approach, having a non framework flavor or an extra field on `apple_library` like `allows_framework_packaging`.